### ISSUE TITLE
Cannot load TensorFlow into Julia 0.7 alpha (released version)

### ISSUE THREAD

**Issue by colbec** [malmaud/TensorFlow.jl#399]
On openSUSE Leap 15 using Version 0.7.0-alpha.0 (2018-05-31 00:07 UTC) I have an error when loading tensorflow. My tensorflow is compiled with bazel and the resulting .so files copied to the TensorFlow/deps/usr/bin. This version works fine in released version 0.6 but has an error in 0.7 released version. I have environment vars set for PYTHON to my 3.6 local version and TENSORFLOW pointing at the deps/usr version.

I get:
```
ERROR: LoadError: LoadError: TypeError: in Type{...} expression, expected UnionAll, got typeof(Base.Nullable)
Stacktrace:
 [1] top-level scope at /home/colin/.julia/packages/AutoHashEquals/tDuU/src/AutoHashEquals.jl:81
 [2] include at ./boot.jl:314 [inlined]
 [3] include_relative(::Module, ::String) at ./loading.jl:1071
 [4] include(::Module, ::String) at ./sysimg.jl:29
 [5] include(::String) at /home/colin/.julia/packages/TensorFlow/TizF/src/TensorFlow.jl:2
 [6] top-level scope
 [7] include at ./boot.jl:314 [inlined]
 [8] include_relative(::Module, ::String) at ./loading.jl:1071
 [9] include(::Module, ::String) at ./sysimg.jl:29
 [10] top-level scope
 [11] eval at ./boot.jl:316 [inlined]
 [12] eval(::Expr) at ./client.jl:394
 [13] macro expansion at ./none:3 [inlined]
 [14] top-level scope at ./<missing>:0
in expression starting at /home/colin/.julia/packages/TensorFlow/TizF/src/core.jl:81
in expression starting at /home/colin/.julia/packages/TensorFlow/TizF/src/TensorFlow.jl:182
```
Is this just me or anyone else getting similar?

**Comment by oxinabox** [comment]
We've not merged any changes to allow 0.7 to work.

**Comment by colbec** [comment]
Indeed, only a problem in v 0.7, Thanks.
